emails/management/commands/get_delayed_emails_from_sqs.py

```diff
@@ -0,0 +1,57 @@
+import json
+import logging
+import sys
+
+import boto3
+from botocore.exceptions import ClientError
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from emails.views import (
+    _sns_inbound_logic, validate_sns_header, verify_from_sns
+)
+
+
+logger = logging.getLogger('events')
+
+
+def _verify_and_run_sns_inbound_on_message(message):
+    json_body = json.loads(message.body)
+    verified_json_body = verify_from_sns(json_body)
+    topic_arn = verified_json_body['TopicArn']
+    message_type = verified_json_body['Type']
+    validate_sns_header(topic_arn, message_type)
+    try:
+        _sns_inbound_logic(topic_arn, message_type, verified_json_body)
+        logger.info(f'processed sqs message ID: {message.message_id}')
+        message.delete()
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'AccountSendingPausedException':
```

Comment:
@groovecoder I don't think this is quite the right error code. I think we would be looking for an error code with the word `Throttling` in it: https://aws.amazon.com/blogs/messaging-and-targeting/how-to-handle-a-throttling-maximum-sending-rate-exceeded-error/

If we catch this particular error, we can probably just sleep for 1 second and try again.

Reply:
Ah ... I updated the code to check for either `throttling` or `pause` in the error code to try to catch either case where the error might be temporary. Hmm ... sleeping and trying again would be cool, but I'm feeling a recursive function call coming on. I'll give it a shot though.

Reply:
Okay, I gave it a `time.sleep(1)` and a 2nd try before it logs the error.